### DISCUSSION TITLE
Inconsistency in hidden files highlight

### DISCUSSION THREAD

**Discussion by Silzinc** [nvim-tree/nvim-tree.lua#2891] [Q&A] [answered]
Hello ! The nvim-tree options doc mentions the following paragraph :
```md
*nvim-tree.renderer.highlight_hidden*
Highlight icons and/or names for hidden files (dotfiles) using the
`NvimTreeHiddenFileHL` highlight group.
Requires |nvim-tree.hidden.enable|
Value can be `"none"`, `"icon"`, `"name"` or `"all"`
  Type: `string`, Default `"none"`
  ```
Though, I cannot see the hidden.enable option anywhere, and I cannot find anything with its role. Is it because I haven't looked enough or does this option not exist ? Thanks in advance.

**Comment by alex-courtis** [comment, marked as answer]
~~`:help nvim-tree.renderer.icons.show.hidden`~~

Edit: that option does not exist and is not needed. Updating the doc. Many thanks for raising!
